src/FNA3D_Driver_Vulkan.c

```diff
@@ -1007,6 +1011,22 @@ static inline void PipelineLayoutHashArray_Insert(
 	arr->count += 1;
 }
 
+typedef struct VulkanSwapchainData
+{
+	VkSwapchainKHR swapchain;
+	VkSurfaceKHR surface;
+	VkSurfaceFormatKHR surfaceFormat;
+	VkPresentModeKHR presentMode;
+	VkFormat swapchainFormat;
+	VkComponentMapping swapchainSwizzle;
+	VkExtent2D extent;
+	VkImage *images;
+	VulkanResourceAccessType *resourceAccessTypes;
+	VkImageView *views;
+	uint32_t imageCount;
+	void *windowHandle;
```

Comment:
Cleaning this up as best as I can...
```suggestion
	/* Window surface */
	VkSurfaceKHR surface;
	VkSurfaceFormatKHR surfaceFormat;
	void *windowHandle;

	/* Swapchain for window surface */
	VkSwapchainKHR swapchain;
	VkFormat swapchainFormat;
	VkComponentMapping swapchainSwizzle;
	VkPresentModeKHR presentMode;

	/* Swapchain images */
	VkExtent2D extent;
	VkImage *images;
	VkImageView *views;
	VulkanResourceAccessType *resourceAccessTypes;
	uint32_t imageCount;
```